Content.Shared/Gibbing/Systems/GibbingSystem.cs

```diff
@@ -152,11 +153,12 @@ public bool TryGibEntityWithRef(
             {
                 foreach (var container in validContainers)
                 {
-                    foreach (var ent in container.ContainedEntities)
+                    var entities = container.ContainedEntities.ToList();
+                    foreach(var ent in entities)
                     {
                         GibEntity(new Entity<GibbableComponent?>(ent, null), parentXform, randomSpreadMod,
-                            ref droppedEntities, launchGibs,
-                            launchDirection, launchImpulse, launchImpulseVariance, launchCone);
+                        ref droppedEntities, launchGibs,
+                        launchDirection, launchImpulse, launchImpulseVariance, launchCone);
```

Comment:
```suggestion
                            ref droppedEntities, launchGibs,
                            launchDirection, launchImpulse, launchImpulseVariance, launchCone);
```
Nitpick, but this is improper indentation